aten/src/ATen/test/extension_backend_test.cpp

```diff
@@ -27,8 +27,9 @@ Tensor empty_override(IntArrayRef size, c10::optional<ScalarType> dtype, c10::op
 }
 
 Tensor add_override(const Tensor & a, const Tensor & b , const Scalar& c) {
+  auto out = empty({5, 5}, at::kMSNPU);  // Don't return self as-is
   test_int = 2;
-  return a;
+  return out;
```

Comment:
Why is a.clone() not enough here? Is clone not implemented for this backend?

Reply:
Nope!